### PR TITLE
Require deme start_time, not epoch start_time.

### PR DESCRIPTION
**Pull request by grahamgower**
Related to #182.

### CODE REVIEW

**Comment by grahamgower** [comment]
Ok, I think this is good to go. It fixes the most invasive things needed to conform to the spec.

File: tests/test_demes.py [hunk 1]
```diff
@@ -1,9 +1,10 @@
 import unittest
 import copy
 import pathlib
-import json
 
-import jsonschema
+# import json
```
Comment (jeromekelleher) [inline tests/test_demes.py:5]: delete?

**Review by jeromekelleher — APPROVED** [review]
LGTM

File: tests/test_demes.py (hunk 1 above)
Comment (grahamgower) [reply tests/test_demes.py]: You're right, it's probably better to just delete this. I thought maybe we'd reuse it again soon for validating against the spec, but probably we'll do something different (via CI) to compare with the spec and/or spec repo examples.

**Pull request merged**